libbpf-tools/kprobe_helpers.cc

```diff
@@ -0,0 +1,178 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/* Copyright (c) 2021 Chenyue Zhou */
+#include <set>
+#include <algorithm>
+#include <string>
+#include <regex>
+#include <fstream>
+#include <sstream>
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "kprobe_helpers.h"
+
+using namespace std;
```

Comment:
`using namespace std` is not a good practice.

IMO, the impl here is not so idiomatic in C++ :)

Reply:
fixed

Reply:
I mean the entire file. Especially `char ***`. With C++, we have better option.

Reply:
Can you give me some advice? I'm not good at C++.